src/applications/disability-benefits/all-claims/tests/schema/minimal-test.json

```diff
@@ -74,62 +51,8 @@
     },
     "view:hasMilitaryRetiredPay": false,
     "view:hasAlternateName": false,
-    "ratedDisabilities": [
-      {
-        "name": "First Condition",
-        "ratedDisabilityId": "0",
-        "ratingDecisionId": "63655",
-        "diagnosticCode": 5238,
-        "decisionCode": "SVCCONNCTED",
-        "decisionText": "Service Connected",
-        "ratingPercentage": 100,
-        "disabilityActionType": "INCREASE"
-      },
-      {
-        "name": "Second Condition",
```

Comment:
Good call. I just deleted a comment I wrote thinking this was the maximal test. :stuck_out_tongue: